schedule/yast/maintenance/register_image.yaml

```diff
@@ -0,0 +1,9 @@
+name: register_image_for_migration_mu_jobs
```

Comment:
`mu` it is a Yam abbreviation, we should not use it anywhere in the code.

Reply:
OK, then how about update `mu` to `maintenance_updates`? or any other suggestions? 
And I named one of the yaml file as `migration_to_mu_unrelease.yaml`, so how about rename it as `migration_to_maintenance_unreleased_updates.yaml`